gobblin-service/src/main/java/org/apache/gobblin/service/modules/spec/JobExecutionPlan.java

```diff
@@ -165,6 +171,8 @@ private static void addAdditionalConfig(JobSpec jobSpec, Config sysConfig) {
 
       Config config = jobSpec.getConfig().withFallback(ConfigUtils.getConfigOrEmpty(sysConfig, additionalConfigsPrefix));
 
+      config = config.withFallback(ConfigUtils.getConfigOrEmpty(specExecutorConfig, additionalConfigsPrefix));
```

Comment:
Will the prefix key be included in the config or will it be dropped? e.g. "specExecutor.a.b" and prefix is "a", will this return "a.b" or just "b"?

Reply:
It will be dropped, i.e. will return b